### DISCUSSION TITLE
Passwörter ändern

### DISCUSSION THREAD

**Discussion by M-Plu** [camueller/SmartApplianceEnabler#364] [Allgemein]
Hallo,

mit der Installation des Images on SAE werden ja auch die zwei Standarduser pi und sae mit Standardkennwörtern angelegt. Kann man diese Kennwörter ändern, oder hat das weitergehende Auswirkungen, die man bedenken müsste, z. B. Konfigurationsanpassungen?

Wenn man die Kennwörter ändern kann, wo macht man das am besten? 

Als Linux unerfahrener User habe ich zunächst auf Webmin geschaut (nach Anmeldung mit User sae) und dort unter "System > Users and Groups" die beiden User gefunden. Klickt man einen der User an, dann ist dort in den Details ein "Pre-encrypted password" hinterlegt. Kann man stattdessen in dem Feld darüber ein "Normal password" setzen und diese Option aktivieren und hat damit die Kennwörter geändert?

**Comment by camueller** [comment]
Der User "pi" wird nicht von der SAE-Installation angelegt, sondern ist vorher schon da.

Die Passwörter der User "pi" und "sae" kannst Du beliebig ändern.

Ich habe noch nie ein Passwort via `webmin` geändert, insofern kann ich Deine Frage dazu nicht beantworten. Du kannst Dich aber einfach per SSH einloggen und `passwd` eingeben, dann wird das Passwort des aktuellen Users geändert.

Vieleicht könntest Du nochmal überlegen, ob Du die Passwörter wirklich ändern willst. Der Raspi befindet sich ja in Deinem (W)LAN, wo vermutlich nur vertrauenswürdige Leute unterwegs sind.

**Reply by M-Plu** [reply]
Vielen Dank für die Rückmeldung. Ich überlege noch mal.
